kotlin/hello-world-lambda-cdk/lambda/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+    <Appenders>
+        <Lambda name="Lambda" format="${env:AWS_LAMBDA_LOG_FORMAT:-TEXT}">
+            <LambdaTextFormat>
+                <PatternLayout>
+                    <pattern>%d{yyyy-MM-dd HH:mm:ss} %notEmpty{%X{AWSRequestId} }%notEmpty{[%X{restateServiceMethod}]}%notEmpty{[%X{restateInvocationId}]} %-5p %c - %m%n - customId=%X{customId}</pattern>
```

Comment:
The customId i guess you don't need it anymore

Reply:
Doh, thanks for catching this!